packages/web/docs/src/components/slider.tsx

```diff
@@ -80,9 +80,11 @@ export function Slider({ counter, className, style, ...rest }: SliderProps) {
           }
         }
 
-        div,
-        div:after {
-          transition: transform var(--ease) 500ms;
+        @supports not (font: -apple-system-body) {
```

Comment:
This detects Safari, not MacOS. `-apple-system-body` is a "precursor" system font.

<img width="300" alt="image" src="https://github.com/user-attachments/assets/9a27b5e2-1183-4c99-9bbd-8031ce574612">
<img width="300" alt="image" src="https://github.com/user-attachments/assets/90d5749f-3833-495b-a743-59e371cf9480">